assets/js/modules/search-console/components/dashboard/SearchFunnelWidget/index.js

```diff
@@ -83,6 +83,32 @@ const SearchFunnelWidget = ( {
 		} )
 	);
 
+	const analyticsGoalsData = useInViewSelect( ( select ) =>
+		select( MODULES_ANALYTICS ).getGoals()
+	);
```

Comment:
Let's also check whether the Analytics module is connected before fetching goals.
```js
const analyticsGoalsData = useInViewSelect( ( select ) =>
	isAnalyticsConnected ? select( MODULES_ANALYTICS ).getGoals() : {}
);
```